### ISSUE TITLE
l11n, i18n

### ISSUE THREAD

**Issue by Vivraan** [Vivraan/godot-lang-support#16]
Let's welcome translation efforts for our document!

Any and all translation effort is welcome, although I believe the following should receive priority first:
1. French
1. Spanish
1. Russian
1. Simplified Chinese
1. German

I myself can translate Hindi and Indian Bengali, although they don't strictly need immediate translation since most Indians using Godot understand English.

**Comment by pycbouh** [comment]
I can give you Russian, but how would that look in the repository? A subfolder for all translation? A `README.ru.md` file?

**Comment by Vivraan** [comment]
Exactly - `README-ru.md` should do!

**Comment by ShalokShalom** [comment]
I could do German. While lets settle first, lets say we wait a little bit, then define a version 0.20 or so and then start translating. Otherwise, we constantly translate already translated stuff, since we change so much underneath. 
:D

**Comment by Vivraan** [comment]
True, we'll at least need a stable English version before translating. Besides, changes to one version would also need to be reflected in every translation.

**Comment by Vivraan** [comment]
I suppose we can begin this after we make a Language Feature Matrix as suggested by @ShalokShalom in #13.
